list/list.go

```diff
@@ -602,7 +602,6 @@ func (m *Model) updateKeybindings() {
 		m.KeyMap.ClearFilter.SetEnabled(false)
 		m.KeyMap.CancelWhileFiltering.SetEnabled(true)
 		m.KeyMap.AcceptWhileFiltering.SetEnabled(m.FilterInput.Value() != "")
-		m.KeyMap.Quit.SetEnabled(true)
```

Comment:
Shouldn't this be here to disable the `Quit` key binding during filtering? (calling it with `false` instead of `true`)

Reply:
This will disable the quit keymap, when it's enabled, after filtering. A better approach could be to store the keymap enabled value, disable it when "Filtering", and set it again after.